From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Tue, 4 Oct 2016 21:26:39 +0200
Subject: [PATCH] {PRINTF} drivers/usb/host/whci: silent -Wformat-security
 warning

clang reports:

    drivers/usb/host/whci/init.c:68:19: error: format string is not a
    string literal (potentially insecure) [-Werror,-Wformat-security]
        whc->workqueue = alloc_ordered_workqueue(dev_name(&whc->umc->dev), 0);
                         ^                       ~~~~~~~~~~~~~~~~~~~~~~~~
---
 drivers/usb/host/whci/init.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/host/whci/init.c b/drivers/usb/host/whci/init.c
index ad8eb575c30a..b55b722201a7 100644
--- a/drivers/usb/host/whci/init.c
+++ b/drivers/usb/host/whci/init.c
@@ -65,7 +65,7 @@ int whc_init(struct whc *whc)
 	init_waitqueue_head(&whc->cmd_wq);
 	init_waitqueue_head(&whc->async_list_wq);
 	init_waitqueue_head(&whc->periodic_list_wq);
-	whc->workqueue = alloc_ordered_workqueue(dev_name(&whc->umc->dev), 0);
+	whc->workqueue = alloc_ordered_workqueue("%s", 0, dev_name(&whc->umc->dev));
 	if (whc->workqueue == NULL) {
 		ret = -ENOMEM;
 		goto error;
-- 
